### PR TITLE
allow anemic tenants

### PR DESCRIPTION
**Pull request by r14chandra**

### CODE REVIEW

File: src/puptoo/app.py [hunk 1]
```diff
@@ -111,10 +111,7 @@ def main():
                     service = service.decode("utf-8")
                     if service in ['advisor', 'compliance', 'malware-detection']:
                         msg = json.loads(msg.value().decode("utf-8"))
-                        # TODO: remove this check when inventory is ready for
```
Comment (kgaikwad) [inline src/puptoo/app.py:114]: As per the comment, I can see that inventory side change from ticket [ESSNTL-2528](https://issues.redhat.com/browse/ESSNTL-2528) is under merged and available for testing.

@kruai - could you please confirm - can we go ahead with this change?

Comment (kruai) [reply src/puptoo/app.py:114]: @kgaikwad There's no description on this PR. What will this do?
If this makes it so "account" isn't always sent, it should wait until [ESSNTL-2529](https://issues.redhat.com/browse/ESSNTL-2529) goes in. It's currently blocked until yupana and rhsm-conduit send org_id for every payload

Comment (kgaikwad) [reply src/puptoo/app.py:114]: > What will this do?

Currently, puptoo checks for `account` value in msg payload. As part of EBS to org_id level 0 & 1 puptoo doesn't allow anemic tenants i.e account value should be present in message.
As part of level 3, with this change service will allow anemic tenants. That means it won't check for `account` value presence.

> If this makes it so "account" isn't always sent, it should wait until ESSNTL-2529 goes in.

With this change, we are just removing `if` condition. Puptoo will still forward `account` value if it is present inside the message payload. 
https://github.com/RedHatInsights/insights-puptoo/blob/master/src/puptoo/mq/msgs.py#L11
But definitely we can wait until ESSNTL-2529 goes in. As 25th July 2022 is decided as a test day on against the stage, we are assuming ESSNTL-2529 will go in before this. Please confirm.

Comment (kruai) [reply src/puptoo/app.py:114]: It should, yes! I just got notified that both remaining reporters might have fixed their org_id issues in Stage, so I'm going to check if I'm able to merge mine. You might be able to merge this sooner than I had anticipated

Comment (kruai) [reply src/puptoo/app.py:114]: @kgaikwad I've just merged my PR, so this one can be merged at any time.

**Review by kruai — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thank you @r14chandra and @kruai for review.

**Pull request merged**